src/main/java/org/broadinstitute/dsde/consent/ontology/translate/service/Translate.java

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.dsde.consent.ontology.translate.service;
+
+import com.github.jsonldjava.shaded.com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import org.broadinstitute.dsde.consent.ontology.translate.DTO.RecommendationDto;
+import org.broadinstitute.dsde.consent.ontology.translate.model.TermItem;
+import org.parboiled.common.FileUtils;
+
+import java.util.HashMap;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class Translate {
+
+    public HashMap<String, RecommendationDto> paragraph(final String paragraph) throws Exception {
+        HashMap<String, RecommendationDto> recommendations = new HashMap<>();
+
+        List<TermItem> terms = loadJSONFromResources("search-terms.json");
```

Comment:
Ontology has a pattern for reading resource files like this. We store them in a bucket and read from that. This lets us modify the file on the fly and bounce any affected pods if necessary. Admittedly, our GCS code is very old in this repo and needs to be completely re-written since Google has updated their java clients. Look for usages of `configurationFileName` which is essentially this file (configurable per environment):
```
{
  "https://storage.googleapis.com/broad-dsde-dev-consent/ontology/disease/diseases.owl": {
    "fileName": "diseases.owl",
    "prefix": "DOID",
    "ontologyType": "Disease"
  },
  "https://storage.googleapis.com/broad-dsde-dev-consent/ontology/organization/data-use.owl": {
    "fileName": "data-use.owl",
    "prefix": "DUOS",
    "ontologyType": "Organization"
  }
}
```
I would rather we maintain the GCS pattern as opposed to the current one in this PR.

Reply:
I still need synonyms.

Reply:
Maybe my suggestion wasn't clear ... I'm not suggesting that you use the actual file I quoted. I'm suggesting that we move `search-terms.json` to the same GCS bucket where we store `configuration.json` (which is here: https://console.cloud.google.com/storage/browser/broad-dsde-dev-consent/ontology). Then use the GCS service class to load this file up instead of reading it from `resources`.

Reply:
Ah, ok, that makes sense then.